collect_app/src/main/java/org/odk/collect/android/formlists/blankformlist/BlankFormListActivity.kt

```diff
@@ -0,0 +1,144 @@
+package org.odk.collect.android.formlists.blankformlist
+
+import android.content.Intent
+import android.net.Uri
+import android.os.Bundle
+import android.view.Menu
+import android.view.MenuItem
+import android.view.View
+import android.widget.ProgressBar
+import android.widget.TextView
+import androidx.activity.viewModels
+import androidx.recyclerview.widget.RecyclerView
+import org.odk.collect.android.R
+import org.odk.collect.android.activities.FormEntryActivity
+import org.odk.collect.android.activities.FormMapActivity
+import org.odk.collect.android.injection.DaggerUtils
+import org.odk.collect.android.network.NetworkStateProvider
+import org.odk.collect.android.preferences.dialogs.ServerAuthDialogFragment
+import org.odk.collect.android.utilities.ApplicationConstants
+import org.odk.collect.android.utilities.SnackbarUtils
+import org.odk.collect.androidshared.ui.DialogFragmentUtils
+import org.odk.collect.permissions.PermissionListener
+import org.odk.collect.permissions.PermissionsProvider
+import org.odk.collect.strings.localization.LocalizedActivity
+import javax.inject.Inject
+
+class BlankFormListActivity : LocalizedActivity(), OnFormItemClickListener {
+
+    @Inject
+    lateinit var viewModelFactory: BlankFormListViewModel.Factory
```

Comment:
It's not a big deal, but it would be nice to avoid adding things to `AppDependencyModule` unless we really need them.

Reply:
Yeah but `BlankFormListViewModel.Factory` takes a lot of things in constructor which I would have to inject here anyway so it would be even worse than having this one field.

Reply:
Yeah, true. OK happy to go with it the way it is right now!